src/sagemaker_inference/transformer.py

```diff
@@ -30,23 +30,25 @@ def find_spec(module_name):
             return None
 
 
-import importlib
+import importlib  # pylint: disable=ungrouped-imports
```

Comment:
it's probably worth revisiting this later. there's a `six.moves.http_client` that probably could replace ll. 15-18. not entirely sure what's going on in ll. 20-30